django/core/exceptions.py

```diff
@@ -71,6 +71,11 @@ class RequestAborted(Exception):
     pass
 
 
+class BadRequest(Exception):
+    """The request is bad and cannot be processed."""
```

Comment:
```suggestion
    """The request is malformed and cannot be processed."""
```

Just the "bad" making me wince a little.